docs/mosviz/import_data.rst

```diff
@@ -67,6 +67,8 @@ and for NIRISS:
 
     jdaviz mosviz /path/to/my/data --instrument=niriss
 
+When loading data from the command line, specifying a data file is optional.
```

Comment:
Do you mean directory?

Reply:
Oh yeah I guess we can only load directories from the cli right now. May as well change that over. Let me fix that

Reply:
Hold up, I think this is actually wrong. I think we actually do require a directory for cli loading... let me check

Reply:
Mosviz indeed requires data from the cli. Verified and fixed. Thanks for the catch!